CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt

```diff
@@ -138,16 +139,17 @@ class GalleryFragment internal constructor() : Fragment() {
                             mediaFile.delete()
 
                             // Send relevant broadcast to notify other apps of deletion
-                            MediaScannerConnection.scanFile(
-                                    view.context, arrayOf(mediaFile.absolutePath), null, null)
+                            MediaScannerConnection.scanFile(view.context,
+                                    arrayOf(mediaFile.absolutePath), null, null)
```

Comment:
Please change this back to the previous form, which is more appropriate as per Kotlin style guides. My interpretation is that you should either put all arguments of a function into a single line (same line as the function name, or the next line) otherwise each argument should be in its own line.